anxcloud/data_source_vlans.go

```diff
@@ -22,7 +22,7 @@ func dataSourceVLANsRead(ctx context.Context, d *schema.ResourceData, m interfac
 	c := m.(providerContext).legacyClient
 	v := vlan.NewAPI(c)
 
-	vlans, err := v.List(ctx, d.Get("page").(int), d.Get("limit").(int), d.Get("search").(string))
+	vlans, err := v.List(ctx, 1, 0, d.Get("search").(string))
```

Comment:
I guess you validated the Engine API really giving everything for `limit=0` for all the resources you use this way?

Reply:
Unfortunately it isn't documented that the Engine API supports 0 Limit to retrieve all resources. But it seems to be working. Should we rather loop with a set limit to be certain that all resources have been fetched?

Reply:
I'd sleep better if you could try to get that info via ENGSUP ticket

Reply:
sadly what I expected